tasks/component-usage-graph.js

```diff
@@ -53,56 +54,59 @@ export async function getComponentsGraph () {
 
 /**
  *
- * @param level
- * @returns {string}
+ * @param {number} level
+ * @returns {string} Returns a string with the correct number of spaces and `│` dividers.
  *
- * Returns a string with the correct number of spaces and `│` dividers.
  */
 function getDividers (level) {
   let dividers = '│';
+
   for (let i = 1; i < level; i++) {
     dividers += `  │ `;
-
   }
+
   return dividers;
 }
 
 /**
  *
- * @param componentPath
- * @param graph
- * @param maxLevel
+ * @param {string} componentPath
+ * @param {DepGraph} graph
+ * @param {number} [maxLevel=Infinity]
  * @returns {string}
  *
+ * This function differs from `getComponentsTree` for various reasons:
+ * * It only takes a component path as a param instead of a list of components paths
+ * * It forms a specific `npm ls` like output
```

Comment:
In JSDoc, the description of the function is always above @param and @return tags.